app/models/manageiq/providers/vmware/cloud_manager/event_catcher/stream.rb

```diff
@@ -12,15 +12,11 @@ def self.test_amqp_connection(options = {})
       connection.start
       return true
     rescue Bunny::AuthenticationFailureError => e
-      _log.info("Failed testing rabbit amqp connection: #{e.message}")
-      raise MiqException::MiqInvalidCredentialsError.new "Login failed due to a bad username or password."
-    rescue Bunny::TCPConnectionFailedForAllHosts => e
-      raise MiqException::MiqHostError.new "Socket error: #{e.message}"
+      raise MiqException::MiqInvalidCredentialsError.new "bad username or password for #{options[:hostname]}"
+    rescue Bunny::TCPConnectionFailed, Bunny::TCPConnectionFailedForAllHosts
+      raise MiqException::MiqHostError.new "cannot reach #{options[:hostname]}:#{options[:port]}"
     rescue
-      _log.info("Failed testing rabbit amqp connection for #{options[:hostname]}. ")
       raise
-    ensure
-      connection.close if connection.respond_to? :close
```

Comment:
What I'm doing here is I'm prettifying the message that user gets when she validates with incorrect hostname or username/password. The `ensure` part had to be removed since it timeout-ed attempting to close a not-even-opened connection. Then user was always given message about timeout even if the problem was actually incorrect password :grimacing: